### PR TITLE
Auxiliary databases for snapshots, CBs and TXs

### PR DESCRIPTION
**Pull request by marcinwadon**
#233

### CODE REVIEW

File: src/main/scala/org/constellation/consensus/EdgeProcessor.scala [hunk 1]
```diff
@@ -59,33 +62,34 @@ object EdgeProcessor extends StrictLogging {
...
           )
         }
-        dao.messageService.put(m.signedMessageData.hash,
+        dao.messageService.putSync(m.signedMessageData.hash,
                                ChannelMessageMetadata(m, Some(cb.baseHash)))
         dao.metrics.incrementMetric("messageAccepted")
       }
 
       // Accept transactions
       cb.transactions.foreach { t =>
```
Comment (Nikolaj-K) [inline src/main/scala/org/constellation/consensus/EdgeProcessor.scala]: It's `tx` in some places in the code and that might be clearer than just `t`.

File: src/main/scala/org/constellation/API.scala [hunk 2]
```diff
@@ -124,7 +124,7 @@ class API()(implicit system: ActorSystem, val timeout: Timeout, val dao: DAO)
               complete(ChannelUIOutput(dao.threadSafeMessageMemPool.activeChannels.keys.toSeq))
             } ~
               path("channel" / Segment / "info") { channelId =>
-                complete(dao.channelService.get(channelId).map { cmd =>
+                complete(dao.channelService.getSync(channelId).map { cmd =>
```
Comment (tylerprete) [inline src/main/scala/org/constellation/API.scala]: Why makes all these sync? akka-http can handle async bodies

Comment (tylerprete) [reply src/main/scala/org/constellation/API.scala]: Though I guess we talked about this -- and it's just swaydb which is super fast. so not a big deal

Comment (marcinwadon) [reply src/main/scala/org/constellation/API.scala]: I've changed names only. Originally there was `get` only which was synchronous. I've changed `get` to async but added `getSync` to maintain backward compatibility without changing whole codebase (but incrementally during working on sth else)

**Pull request merged**